cpu/cortexm_common/thread_arch.c

```diff
@@ -234,6 +238,17 @@ char *thread_arch_stack_init(thread_task_func_t task_func,
     return (char*) stk;
 }
 
+/* This function gets the number of bytes used on the ISR stack */
+int thread_arch_isr_stack_usage(void)
+{
+    int count = 0;
+    uint32_t* sp = (uint32_t *) _estack;
+    while(sp != (uint32_t *) _sstack)
+        if (*sp == STACK_CANARY_WORD)
+            count++;
+    return ((int)(512 - count));
```

Comment:
Yes, it does imply this. But this is valid, as this is the way stacks are handled on the Cortex-M platforms -> http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0553a/BABDGADF.html